src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -24,6 +25,10 @@ def call
 
   private
 
+  def modify_target_project_url
+    target_project.update(url: workflow_run.event_source_url)
```

Comment:
Unlike `Package`, `Project` has to be explicitly written to the backend...

Reply:
True. Please take a look again.